integration-tests/src/test/resources/features/Evaluation.feature

```diff
@@ -1,22 +1,44 @@
 Feature: Evaluation of Publications as NVI Candidates
-  Rule: An applicable Publication in an open Period is a Candidate
-    Scenario: A new Publication is evaluated
+
+  Rule: Publications can become Candidates if the Period is not closed
+    Scenario Outline: A new applicable Publication becomes a Candidate
       Given an applicable Publication
-      And the reporting period for the Publication is "OPEN"
+      And the reporting period for the Publication is "<period_state>"
       When the Publication is evaluated
-      Then the Publication is a Candidate
+      Then the Publication is persisted as a Candidate
 
-  Rule: An applicable Candidate re-evaluated in a closed Period is a Candidate
-    Background:
-      Given a Publication that has previously been evaluated as a Candidate
+      Examples:
+        | period_state |
+        | OPEN         |
+        | PENDING      |
 
-    Scenario: An applicable Publication remains a Candidate
-      Given the reporting period for the Publication is "CLOSED"
+  Rule: Unreported Candidates can be updated regardless of period state
+    Scenario Outline: An applicable Publication remains a Candidate
+      Given a Publication that has previously been evaluated as a Candidate
+      And the Candidate is not reported
+      And the reporting period for the Publication is "<period_state>"
       When the Publication is evaluated
-      Then the Publication is a Candidate
+      Then the Publication is persisted as a Candidate
+      And the persisted data is updated
+
+      Examples:
+        | period_state |
+        | OPEN         |
+        | PENDING      |
+        | CLOSED       |
 
-    Scenario: A non-applicable Publication becomes a NonCandidate
-      Given the reporting period for the Publication is "CLOSED"
+    Scenario Outline: A non-applicable Publication becomes a NonCandidate
+      Given a Publication that has previously been evaluated as a Candidate
```

Comment:
Does this:
`Given a Publication that has previously been evaluated as a Candidate`
mean the same as:
`Given a nvi candidate`?

Can whole gherkin be simplified to something like:

```
Scenario: Candidate becomes NonCandidate when publication type becomes non-applicable
  Given an unreported NVI Candidate
  When the publication type changes to a non-applicable type
  And the publication is re-evaluated
  Then it is persisted as a NonCandidate
```

Reply:
Kind of. It's verbose because I was trying to differentiate between "current state of the publication (upstream service)" and "what we have persisted as publication metadata (in this service)". You can change the publication type of a publication, but the publication type persisted for the corresponding candidate won't be updated if the candidate is already reported.

I'll try to make this more concise and still accurate.